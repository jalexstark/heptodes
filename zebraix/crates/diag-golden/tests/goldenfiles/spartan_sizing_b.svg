<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="300pt" viewBox="0 0 400 300">
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 32 252 L 32 12 M 352 252 L 352 12 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.5" stroke-miterlimit="10" d="M 32 12 L 352 252 M 352 12 L 32 252 "/>
</svg>
