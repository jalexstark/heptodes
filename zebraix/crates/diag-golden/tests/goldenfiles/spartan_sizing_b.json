{
  "sizing_scheme": "Fill",
  "axes_range": [
    2.0
  ],
  "padding": [
    0.1,
    0.2,
    0.15,
    0.05
  ]
}
